src/integration-test/java/com/commercetools/project/sync/ProductSyncWithReferencesIT.java

```diff
@@ -67,8 +78,30 @@
 class ProductSyncWithReferencesIT {
 
   private static final TestLogger syncerTestLogger = TestLoggerFactory.getTestLogger(Syncer.class);
```

Comment:
Do you think we can remove it? Since there is no logger object from Syncer anymore based on my understanding.

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/264/commits/b16552ac3b3e6f10bf22b3b71a69049685a48f4e